website/settings/defaults.py

```diff
@@ -79,11 +79,16 @@ def parent_dir(path):
 SHARE_ELASTIC_INDEX_TEMPLATE = 'share_v{}'
 
 # Sessions
+COOKIE_NAME = 'osf'
 # TODO: Override OSF_COOKIE_DOMAIN in local.py in production
 OSF_COOKIE_DOMAIN = None
-COOKIE_NAME = 'osf'
+# TODO: Override OSF_COOKIE_SECURE in local.py in production
+COOKIE_SECURE = False
```

Comment:
Might be better to rename this to something more generic, so that one setting controls all behavior required to turn on SSH for local development.